core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -712,6 +746,184 @@ export class LoggedInUser extends BaseUser {
+    );
+    await this.clickOn(dismissWelcomeModalSelector);
+    await this.page.waitForSelector(dismissWelcomeModalSelector, {
+      hidden: true,
+    });
+    await this.clickOn(textStateEditSelector);
+    await this.page.keyboard.press('Tab');
+    await this.type(richTextAreaField, 'Test Exploration');
+    await this.clickOn(saveContentButton);
+
+    await this.clickOn(addInteractionButton);
+    await this.page.waitForSelector('oppia-customize-interaction', {
+      visible: true,
+    });
+    await this.clickOn(interactionEndExplorationInputButton);
+    await this.clickOn(saveInteractionButton);
+
+    if (this.isViewportAtMobileWidth()) {
+      await this.page.click(mobileExplorationEditorOptions);
+      await this.clickOn(saveDraftMobileButton);
+      await this.clickOn(saveDraftButton);
+      await this.page.waitForSelector(toastMessage, {visible: true});
+      await this.page.waitForSelector(toastMessage, {visible: false});
+      await this.clickOn(explorationMobileDropdown);
+      await this.clickOn(publishExplorationMobileButton);
+    } else {
+      await this.clickOn(saveChangesButton);
+      await this.clickOn(saveDraftButton);
+      await this.page.waitForSelector(
+        `${publishExplorationButton}:not([disabled])`
+      );
+      await this.clickOn(publishExplorationButton);
+    }
+
+    await this.type(explorationTitleInput, 'Test Exploration');
+    await this.type(explorationGoalInput, 'Test Exploration');
+    await this.clickOn(explorationCategoryDropdown);
+    await this.clickOn('Algebra');
+    await this.clickOn(saveExplorationChangesButton);
+    await this.clickOn(explorationConfirmPublishButton);
+    await this.page.waitForSelector(explorationIdElement);
+    await this.page.click('.e2e-test-share-publish-close');
+    const explorationIdUrl = await this.page.$eval(
+      explorationIdElement,
+      element => element.textContent
+    );
+    return explorationIdUrl;
+  }
+
+  /**
+   * Function for navigating to the profile page for a given username.
+   */
+  async navigateToProfilePage(username: string): Promise<void> {
+    const profilePageUrl = `${profilePageUrlPrefix}/${username}`;
+    if (this.page.url() === profilePageUrl) {
+      return;
+    }
+    await this.goto(profilePageUrl);
+  }
+
+  /**
+   * Function to subscribe to a creator with the given username.
+   */
+  async subscribeToCreator(username: string): Promise<void> {
+    const profilePageUrl = `${profilePageUrlPrefix}/${username}`;
+
+    if (this.page.url() !== profilePageUrl) {
+      await this.navigateToProfilePage(username);
+    }
+
+    await this.clickOn(subscribeButton);
+    await this.page.waitForSelector(unsubscribeLabel);
+    showMessage(`Subscribed to the creator with username ${username}`);
+  }
+
+  /**
+   * This function checks if the number of subscribers equal to.
+   */
+  async expectNumberOfSubscribersToBe(number: number): Promise<void> {
```

Comment:
Name the parameter `subscriberCount`. We want to know *what* the function argument will represent, not *how* it will be represented as. The type system will tell us how the function argument will be represented as (in this case, it will be a `number`)

Reply:
Done.